src/components/OpportunityCard/OpportunityCard.tsx

```diff
@@ -198,7 +198,12 @@ export const OpportunityCard = ({
           className={`flex flex-wrap-reverse items-center gap-x-7 gap-y-5 mb-4 order-1 ${typeof matchValue === 'number' && matchLabel ? 'justify-between' : 'justify-end'}`}
         >
           {typeof matchValue === 'number' && matchLabel && (
-            <div className="flex flex-nowrap gap-x-3 items-center px-4 bg-[#AD4298] rounded-full text-white select-none">
+            <div
+              className={cx('flex flex-nowrap gap-x-3 items-center px-4 text-white rounded-full select-none', {
+                'bg-[#AD4298]': type === 'TYOMAHDOLLISUUS',
+                'bg-[#00818A]': type === 'KOULUTUSMAHDOLLISUUS',
```

Comment:
These same colors are used in Tool/Competences and other also in Tool/Interests.

Should we make tokens from these colors as there are no existing tokens for these? 🤔

Reply:
I guess it would be a good idea, but maybe we'll go with hex values for now, since it's only 2 places. Naming should be thought of too, I don't think we need any more `secondary-*` colors 😅

I also keep wondering why the guys who make the Figma designs do not use their own design tokens. I think this should be addressed in the next PAMU session.

Reply:
Yeah, these hex values can be looked also later from the code and create the tokens if needed.

It's good idea to bring the tokens use back to table in the next PAMU session; it's same for color and typography tokens.